recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/doc/bblayers.conf.dom0-image-minimal-initramfs

```diff
@@ -0,0 +1,21 @@
+# LAYER_CONF_VERSION is increased each time build/conf/bblayers.conf
+# changes incompatibly
+LCONF_VERSION = "6"
```

Comment:
what is `LCONF_VERSION`? You didn't mention it in comment. And I can't find it in Yocto Reference Manual.

Reply:
typo, fixed